### PR TITLE
Adds getValueForBbox

### PR DESCRIPTION
**Pull request by fdansv**
Adds a function that returns the accumulated value of the bounding box specified.
peek

@saleiva 
@javisantana

### CODE REVIEW

**Comment by saleiva** [comment]
Does it work for heatmaps too?

**Comment by fdansv** [comment]
Yep (it does now, in the prev. example I was only taking measures for the first row...)

peek_heat

File: lib/torque/leaflet/torque.js [hunk 1]
```diff
@@ -384,6 +384,27 @@ L.TorqueLayer = L.CanvasLayer.extend({
     return null;
   },
 
+  getValueForBBox: function(x, y, w, h) {
+    var xf = x + w, yf = y + h, _x=x;
+    var sum = 0;
+    while(y<yf){
```
Comment (javisantana) [inline lib/torque/leaflet/torque.js]: why not a for loop? :tmcw:

**Comment by javisantana** [comment]
really nice, i miss:

- google maps support
- NEWS 
- DOC

**Comment by saleiva** [comment]
I'm with santana, we need the NEWS, the doc, and the way to activate this from the actual editor. I'll design that later this week, ok?

**Comment by fdansv** [comment]
Yeah I mean, all the UI you see is something I added to the example. Here I'm just adding the method that makes it possible. UI/editor is for PRs in cartodb.js/cartodb.

**Comment by fdansv** [comment]
Added support for Google Maps, same API :)

**Comment by fdansv** [comment]
retest this please

**Comment by fdansv** [comment]
@javisantana :it: ?

**Pull request merged**